kolibri/plugins/coach/assets/src/views/groups-page/index.vue

```diff
@@ -1,18 +1,20 @@
 <template>
 
   <div>
-    <section>
-      <h1 :class="{header: sortedGroups.length}">{{ $tr('classGroups') }}</h1>
-
-      <span v-if="!sortedGroups.length">{{ $tr('noGroups') }}</span>
-
-      <k-button
-        class="new-group-button"
-        :text="$tr('newGroup')"
-        :primary="true"
-        @click="openCreateGroupModal"
-      />
-    </section>
+    <k-grid>
+      <k-grid-item sizes="100, 50, 50" percentage>
+        <h1>{{ $tr('classGroups') }}</h1>
+        <p v-if="!sortedGroups.length">{{ $tr('noGroups') }}</p>
+      </k-grid-item>
+      <k-grid-item sizes="100, 50, 50" percentage alignment="right">
+        <k-button
+          class="new-group-button"
+          :text="$tr('newGroup')"
+          :primary="true"
+          @click="openCreateGroupModal"
+        />
+      </k-grid-item>
+    </k-grid>
```

Comment:
This is an example of how a basic responsive grid can be specified using percentages. The three size numbers given in each grid item correspond to the widths (in percent) at small, medium, and large window sizes respectively.